app/services/paypal/PayPalBillingDetails.scala

```diff
@@ -0,0 +1,3 @@
+package services.paypal
+
+case class PayPalBillingDetails(amount: Float, billingPeriod: String, currency: String)
```

Comment:
support-internationalisation defines a type `Currency`, could you use it here?

What are the valid values of billingPeriod?